cli/command/configure/cmd.go

```diff
@@ -0,0 +1,142 @@
+package configure
+
+import (
+	"errors"
+	"fmt"
+	"github.com/rs/zerolog/log"
+	"github.com/saucelabs/saucectl/cli/command"
+	"github.com/saucelabs/saucectl/cli/credentials"
+	"github.com/spf13/cobra"
+	"github.com/tj/survey"
+	"os"
+	"regexp"
+)
+
+var (
+	configureUse     = "configure"
+	configureShort   = "Configure your Sauce Labs credentials"
+	configureLong    = `Persist locally your Sauce Labs credentials`
+	configureExample = "saucectl configure"
+	cliUsername      = ""
+	cliAccessKey     = ""
+)
+
+// Command creates the `new` command
+func Command(cli *command.SauceCtlCli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     configureUse,
+		Short:   configureShort,
+		Long:    configureLong,
+		Example: configureExample,
+		Run: func(cmd *cobra.Command, args []string) {
+			log.Info().Msg("Start Configure Command")
+			if err := Run(cmd, cli, args); err != nil {
+				log.Err(err).Msg("failed to execute configure command")
+				os.Exit(1)
+			}
+		},
+	}
+	cmd.Flags().StringVarP(&cliUsername, "username", "u", "", "username,, available on your saucelabs account")
+	cmd.Flags().StringVarP(&cliAccessKey, "accessKey", "a", "", "accessKey, available on your saucelabs account")
+	return cmd
+}
+
+// explainHowToObtainCredentials explains how to get credentials
+func explainHowToObtainCredentials() {
+	fmt.Println("\nDon't have an account ? Signup here https://saucelabs.com/sign-up !")
+	fmt.Printf("Already have an account ? Get your username and access key here: https://app.saucelabs.com/user-settings\n\n\n")
```

Comment:
@FriggaHel should this be using `zerolog` instead of `fmt.Print`?

Reply:
Since it was more related to user interaction than logging, I went using `fmt.Print*`.
I tried with `zerolog` but found no way to have a behaviour which is similar to `survey` to have a coherent display with the `survey` part.
I think it can be done by tweaking the logger output (preventing timestamp / severity from being displayed), but I don't think this is a smart move.